content/en/docs/tasks/administer-cluster/nodelocaldns.md

```diff
@@ -47,18 +47,38 @@ This is the path followed by DNS Queries after NodeLocal DNSCache is enabled:
 {{< figure src="/images/docs/nodelocaldns.jpg" alt="NodeLocal DNSCache flow" title="Nodelocal DNSCache flow" caption="This image shows how NodeLocal DNSCache handles DNS queries." >}}
 
 ## Configuration
-
-This feature can be enabled using the command:
-
-`KUBE_ENABLE_NODELOCAL_DNS=true kubetest --up`
-
-This works for e2e clusters created on GCE. On all other environments, the following steps will setup NodeLocal DNSCache:
-
-* A yaml similar to [this](https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/dns/nodelocaldns/nodelocaldns.yaml) can be applied using `kubectl create -f` command.
-* No need to modify the --cluster-dns flag since NodeLocal DNSCache listens on both the kube-dns service IP as well as a link-local IP (169.254.20.10 by default)
+{{< note >}} The local listen IP address for NodeLocal DNSCache can be any IP in the 169.254.20.0/16 space or any other IP address that can be guaranteed to not collide with any existing IP. We will use 169.254.20.10 as the default in the example below.
+{{< /note >}}
+
+This feature can be enabled using the following steps:
+* Prepare a manifest similar to the sample [`nodelocaldns.yaml`](https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/dns/nodelocaldns/nodelocaldns.yaml) and save it as `nodelocaldns.yaml.`
+* Substitute the variables in the manifest with the right values:
+   kubedns=`kubectl get svc kube-dns -n kube-system -o jsonpath={.spec.clusterIP}`
+   domain="cluster.local"
+   localdns="169.254.20.10"
```

Comment:
I'd rather leave it up to the reader to pick an address, rather than implying that 169.254.20.10 is somehow special.

Reply:
How about using an IPv4 address from https://tools.ietf.org/html/rfc5737?

Reply:
The only reason to put 169.254.20.10 is to provide an example. I have added a note right above this in line 50, saying the IP address can be from any range as long as it is guaranteed to not collide. I can make this "a.b.c.d" if that is better. What do you think? @sftim

Reply:
How about using a documentation address (see [RFC 5737](https://tools.ietf.org/html/rfc5737))?

Reply:
I would prefer to not introduce a new IP address, for documentation. The 169.254 address has appeared in prior docs/slides about this feature and will also be the IP used in some environments. So, it would be less confusing. If we added a 192.0.2.1 or similar address in the doc, it could be confusing as to where it came from.

Reply:
Sounds like `a.b.c.d` works better? OK, that makes sense.
Could also have:
```
   domain="<cluster-domain>"
   localdns="<node-local-address>"
```
and explain _<cluster-domain>_ and _<node-local-address>_ in the text.

@prameshj I think this is close to mergeable - there are a few quibbles that I hope and believe are not major.

Reply:
done